app/javascript/components/analytics/public_health/charts/MonitoreeFlow.js

```diff
@@ -118,17 +158,133 @@ class MonitoreeFlow extends React.Component {
+                {data.map((x, i) => (
+                  <td key={i}>
+                    <div className="count-percent-container">
+                      <span className="number">{x.cases_closed_in_exposure.value}</span>
+                      <span className="percentage">({x.cases_closed_in_exposure.percentage})</span>
+                    </div>
+                  </td>
+                ))}
+              </tr>
+              <tr className="g-border-bottom">
+                <td className="header">Total Contacts that became Cases</td>
+                {data.map((x, i) => (
+                  <td key={i}>
+                    <div className="count-percent-container">
+                      <span className="number">{x.exposure_to_isolation_total.value}</span>
+                      <span className="percentage"></span>
+                    </div>
+                  </td>
+                ))}
+              </tr>
+            </tbody>
+          </table>
+        </div>
+      </Col>
+    );
+  }
+
+  renderIsolationToExposureTable(data) {
+    return (
+      <Col xl="12">
+        <div className="analytics-table-header">Moved from Isolation to Exposure</div>
+        <div className="table-responsive">
+          <table className="analytics-table">
+            <thead>
+              <tr className="g-border-bottom text-center header">
+                <th></th>
+                <th>Yesterday</th>
+                <th>Last 7d</th>
+                <th>Last 14d</th>
+                <th>Cumulative</th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr className="g-border-bottom">
+                <td className="header">Moved from Isolation to Exposure Workflow</td>
+                {data.map((x, i) => (
+                  <td key={i}>
+                    <div className="count-percent-container">
+                      <span className="number">{x.isolation_to_exposure_total.value}</span>
+                      <span className="percentage align-bottom"></span>
+                    </div>
+                  </td>
+                ))}
+              </tr>
+            </tbody>
+          </table>
+        </div>
+      </Col>
+    );
+  }
+
   render() {
     return (
       <Card>
         <Card.Header as="h4" className="text-center">
           Monitoree Flow Over Time (All Records)
         </Card.Header>
         <Card.Body>
-          <Row>{this.tableData.map((data, index) => this.renderWorkflowTable(data, index))}</Row>
-          <div className="text-center text-secondary info-text my-1">
+          <Row>{this.workflowTableData.map((data, index) => this.renderWorkflowTable(data, index))}</Row>
+          <div className="text-center text-secondary info-text mb-4">
+            <i className="fas fa-info-circle mr-1"></i>
+            Cumulative includes incoming and outgoing counts recorded for this jurisdiction (excluding today’s counts). All counts calculated based on UTC time
```

Comment:
I'm nitpicking, but it feels like the second sentence should have a "." at the end.